README.md

```diff
@@ -22,11 +22,7 @@ To get started with simple deployment, please follow [the getting started guide]
 
```

Comment:
I think Requirements section should also be updated.

Reply:
Do you mean add `scalar-terraform-examples` ?

I tried to fix in c94cb48.

Reply:
No, I thought it does not require terraform any more, doesn't it?